include/aspect/utilities.h

```diff
@@ -123,6 +123,14 @@ namespace aspect
     }
 
     /**
+     * Given a 2d point and a list of points which form a polygon, computes if the point
+     * falls within the polygon.
+     */
+    template <int dim>
+    bool
+    in_2d_polygon(dealii::Point<2> &point,const std::vector<std::vector<double> > &pointList);
```

Comment:
same here -- change it to `vector<Point<2>>`. Make the `point` argument `const`.

For unclear, historical reasons, we seem to usually list the bigger arguments first, so `points_list` (not name) first and `point` second.